src/common/options.cc

```diff
@@ -8089,6 +8089,7 @@ std::vector<Option> get_mds_options() {
 
     Option("mds_bal_export_pin", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
     .set_default(true)
+    .set_flag(Option::FLAG_RUNTIME)
```

Comment:
Thanks for fixing these. If you've converted all uses of a config from `conf->foo` to `conf.get_val<bar>("foo")` then you can also remove the config from `src/common/legacy_config_opts.h`.

Reply:
Ack. Done!